geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/RemoveCommand.java

```diff
@@ -71,30 +69,28 @@ public Result remove(
       cache.getSecurityService().authorize(Resource.DATA, Operation.WRITE, regionPath, key);
     }
 
-    @SuppressWarnings("rawtypes")
     Region region = cache.getRegion(regionPath);
     DataCommandFunction removefn = new DataCommandFunction();
+    DataCommandResult dataResult;
     if (region == null) {
       Set<DistributedMember> memberList = getRegionAssociatedMembers(regionPath, getCache(), false);
-      if (CollectionUtils.isNotEmpty(memberList)) {
-        DataCommandRequest request = new DataCommandRequest();
-        request.setCommand(CliStrings.REMOVE);
-        request.setKey(key);
-        request.setKeyClass(keyClass);
-        request.setRemoveAllKeys(removeAllKeys ? "ALL" : null);
-        request.setRegionName(regionPath);
-        dataResult = callFunctionForRegion(request, removefn, memberList);
-      } else {
-        dataResult = DataCommandResult.createRemoveInfoResult(key, null, null,
-            CliStrings.format(CliStrings.REMOVE__MSG__REGION_NOT_FOUND_ON_ALL_MEMBERS, regionPath),
-            false);
+
+      if (CollectionUtils.isEmpty(memberList)) {
+        return createUserErrorResult("Region <" + regionPath + "> not found in any of the members");
```

Comment:
Why change from using the formatted message in CliStrings to defining the message on the fly here. I favor using a message string defined in one place, to be used in the production code and associated tests, rather than incurring possible test maintenance issues having to modify strings in multiple places if they should change.

Reply:
I think that the huge list of strings in CliStrings is an antipattern that I would like to get away from.  What do you think about keeping this String as a `public static final` constant in this class rather than in CliStrings?

Reply:
Yes, I can accept that; takes care of my biggest concern about having to maintian literal strings in more than one place.